recipes/fury/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "fury" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cbe48366de47074d65491a1c8d6cc3efd917c1013925b7652733538e3db932cd
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - numpy
+    - scipy
+    - vtk
+
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
```

Comment:
As you are not compiling against ``numpy`` the pin here should not be necessary

https://conda-forge.org/docs/meta.html#building-against-numpy